# Project 1: Tuberculosis Deaths among BRICS and Portuguese-speaking countries

by Afyia Miller, 15 Nov 2017

Per TB Alliance Tuberculosis is a global pandemic, killing someone approximate every 18 seconds. Worldwide Tuberculosis (TB) is ranked as the ninth leading cause of death and the leading cause of death among an infections such as HIV/AIDS. In 2016, there were an estimated 10.4 million new cases (incidence) of TB infections and 1.3 million attributed to deaths among the HIV-seronegative population. Among the 10.4 million new cases of TB in 2016, 10% were HIV positive and 56% lived in five countries such as India and China. As a result of the high incidence, India and China will be included in the analysis of BRICS (Brazil, Russia, India, China and South Africa) countries.

The objective of this analysis is to explore the following questions:

1). What is the total, average and range of TB deaths in 2013?
2). Which BRICS countries have the most and least deaths?
3). What is the death rate (deaths per 100,000 persons) for each country?
4). Which countries have the lowest and highest death rate?

** The death rate allows countries with varying population sizes to be compared **


# Data Source

The data consists of total population and total number of deaths due to TB among a non-HIV population in 2013 among BRICS  and Portuguese-speaking countries.

The data was taken in July 2015 from http://apps.who.int/gho/data/node.main.POP107?lang=en (population) and http://apps.who.int/gho/data/node.main.593?lang=en (deaths). The uncertainty bounds of the number of deaths were ignored.
The data was collected into an Excel file which should be in the same folder as this notebook.


In [1]:
import warnings
warnings.simplefilter ('ignore', FutureWarning)

from pandas import * 
data=read_excel('WHO POP TB some.xls')
data

Country  Population (1000s)  TB deaths
0                  Angola               21472       6900
1                  Brazil              200362       4400
2                   China             1393337      41000
3       Equatorial Guinea                 757         67
4           Guinea-Bissau                1704       1200
5                   India             1252140     240000
6              Mozambique               25834      18000
7                Portugal               10608        140
8      Russian Federation              142834      17000
9   Sao Tome and Principe                 193         18
10           South Africa               52776      25000
11            Timor-Leste                1133        990

# Methods

In [2]:
tbColumn = data['TB deaths']

The total number of deaths in 2013 is:

In [3]:
tbColumn.sum()

354715

The Maximun of TB deaths in 2013 is:

In [4]:
 
tbColumn.max()  


240000

The Minimum of TB deaths in 2013 is:

In [5]:
tbColumn.min() 

18

The range of deaths among BRICS and Portuguese-speaking countries range from (18 - 240,000). This is very alarming hence a better approach may be to determine the average number of deaths across all countries in the data, thereby providing a more context to TB incidence. Given the skewness of deaths, median is a better average measure in comparison to the mean.

In [6]:
tbColumn.median()

5650.0

In [7]:
round(tbColumn.mean(),0)

29560.0

Given that the median is significantly lower than the mean indicates that some countries had a very high number of TB deaths in 2013, thereby forcing the value of the mean up.

# What countries have been affected more?

To see the countries that have been affected more, the table was sorted in descending order by TB Deaths, which puts those countries in the top rows.

In [8]:
data.sort_values('TB deaths', ascending=False)

Country  Population (1000s)  TB deaths
5                   India             1252140     240000
2                   China             1393337      41000
10           South Africa               52776      25000
6              Mozambique               25834      18000
8      Russian Federation              142834      17000
0                  Angola               21472       6900
1                  Brazil              200362       4400
4           Guinea-Bissau                1704       1200
11            Timor-Leste                1133        990
7                Portugal               10608        140
3       Equatorial Guinea                 757         67
9   Sao Tome and Principe                 193         18

Reviewing the data suggests that countries with a greater population have a larger number of TB deaths. To further assess and compare these countries equally, the death rate per 100,000 persons is computed.

In [13]:
popColumn = data['Population (1000s)']
data['TB deaths (per 100,000)'] = round((tbColumn/popColumn) *100,2)
data


Country  Population (1000s)  TB deaths  \
0                  Angola               21472       6900   
1                  Brazil              200362       4400   
2                   China             1393337      41000   
3       Equatorial Guinea                 757         67   
4           Guinea-Bissau                1704       1200   
5                   India             1252140     240000   
6              Mozambique               25834      18000   
7                Portugal               10608        140   
8      Russian Federation              142834      17000   
9   Sao Tome and Principe                 193         18   
10           South Africa               52776      25000   
11            Timor-Leste                1133        990   

    TB deaths (per 100,000)  
0                     32.13  
1                      2.20  
2                      2.94  
3                      8.85  
4                     70.42  
5                     19.17  
6                     69.68  
7                      1.32  
8                     11.90  
9                      9.33  
10                    47.37  
11                    87.38

# Conclusions

The BRICS and Portuguese-speaking countries had approximately 350,000 deaths due to TB in 2013 alone. The meadian suggest that half of these countries had 5,650 deaths or less, the mean average of TB deaths was higher at 29560 thereby indicating that some countries had a much higher number of deaths. Sao Tome and Principe as well as Equatorial Guinea had a total of 18 and 67 TB deaths respectively. It should also be noted that the two aforementioned countries also had the lowest population  at 193 and 757 per 1000 persons. China and India contributed to the skewness of the mean with both having 41,000 and 240,000 TB deaths respectively in 2013. As shown in table above China and India both had the largest populations per 1000 persons.

Considering the size of the population the least affected countries were Brazil and Portugal with less than 2.2 deaths per 100,000 persons similarly, Guinea-Bissau and Timor-Leste were the most affected with 70.4 and 87.4 deaths per 100,000 persons.

One limitation to this analysis is the small sample of countries used in the analysis. Further research is needed to assess TB deaths disparity among countries and more interventions are need to elimiate TB as global pandemic.  